src/module/module-collection.js

```diff
@@ -53,6 +53,14 @@ export default class ModuleCollection {
 
     parent.removeChild(key)
   }
+
+  isRegistered (path) {
+    const parent = this.get(path.slice(0, -1))
```

Comment:
But the purpose of this feature is to check existance of module, isn't it? I think the `hasModule` should return `false` in that case.